lib/ts/recipe/accountlinking/types.ts

```diff
@@ -184,15 +177,32 @@ export type RecipeInterface = {
     unlinkAccounts: (input: {
```

Comment:
merge this also (after review): https://github.com/supertokens/supertokens-node/pull/500